### PR TITLE
Translate react developer tools

### PR DESCRIPTION
**Pull request by Dlurak**

### CODE REVIEW

File: src/content/learn/react-developer-tools.md [hunk 1]
```diff
@@ -36,26 +36,27 @@ yarn global add react-devtools
...
 ```
 
-Next open the developer tools from the terminal:
+Öffne als nächstes die developer tools aus dem Terminal:
 ```bash
 react-devtools
 ```
 
-Then connect your website by adding the following `<script>` tag to the beginning of your website's `<head>`:
+Verbinde dann deine Webseite indem du das folgende `<script>` Tag zum Anfang des `<head>` deiner Webseite hinzufügst:
```
Comment (ph1p) [inline src/content/learn/react-developer-tools.md:44]:
```suggestion
Verbinde dann deine Webseite, indem du das folgende `<script>` Tag zum Anfang des `<head>` deiner Webseite hinzufügst:
```

File: src/content/learn/react-developer-tools.md [hunk 2]
```diff
@@ -36,26 +36,27 @@ yarn global add react-devtools
...
 ```html {3}
 <html>
   <head>
     <script src="http://localhost:8097"></script>
 ```
 
-Reload your website in the browser now to view it in developer tools.
+Lade deine Webseite jetzt im Browser neu um die developer tools zu sehen.
```
Comment (ph1p) [inline src/content/learn/react-developer-tools.md:51]:
```suggestion
Lade deine Webseite jetzt im Browser neu, um die Developer-Tools zu sehen.
```

File: src/content/learn/react-developer-tools.md [hunk 3]
```diff
@@ -36,26 +36,27 @@ yarn global add react-devtools
...
 
-![React Developer Tools standalone](/images/docs/react-devtools-standalone.png)
+![Eigenständige React Developer Tools](/images/docs/react-devtools-standalone.png)
 
 ## Mobile (React Native) {/*mobile-react-native*/}
-React Developer Tools can be used to inspect apps built with [React Native](https://reactnative.dev/) as well.
+React Developer Tools kann ebenfalls genutzt werden um Apps, die mit [React Native](https://reactnative.dev/) erstellt wurden, zu inspizieren.
 
 The easiest way to use React Developer Tools is to install it globally:
```
Comment (ph1p) [inline src/content/learn/react-developer-tools.md:58]:
```suggestion
```

File: src/content/learn/react-developer-tools.md [hunk 4]
```diff
@@ -64,13 +65,13 @@ yarn global add react-devtools
 npm install -g react-devtools
 ```
 
-Next open the developer tools from the terminal.
+Öffne als nächstes die developer tools aus dem Terminal:
```
Comment (ph1p) [inline src/content/learn/react-developer-tools.md:68]:
```suggestion
Öffne als Nächstes die Developer Tools aus dem Terminal:
```

File: src/content/learn/react-developer-tools.md [hunk 5]
```diff
@@ -64,13 +65,13 @@ yarn global add react-devtools
...
 ```bash
 react-devtools
 ```
 
-It should connect to any local React Native app that's running.
+Es sollte sich mit jeder lokal laufenden React Native App verbinden.
 
-> Try reloading the app if developer tools doesn't connect after a few seconds.
+> Versuche die App neuzuladen, falls die developer tools sich nicht nach einigen Sekunden verbinden.
```
Comment (ph1p) [inline src/content/learn/react-developer-tools.md:75]:
```suggestion
> Versuche die App neu zuladen, falls die Developer-Tools sich nicht nach einigen Sekunden verbinden.
```

**Review by ph1p — CHANGES_REQUESTED** [review]

**Comment by ph1p** [comment]
Thank you :) I added some suggestions

File: src/content/learn/react-developer-tools.md (hunk 3 above)
Comment (Dlurak) [reply src/content/learn/react-developer-tools.md:58]: Oops, I've overlooked this line :sweat_smile:

**Comment by Dlurak** [comment]
Wow, thanks for fixing all of these errors.

**Pull request merged**